libpod/runtime.go

```diff
@@ -692,13 +692,13 @@ func makeRuntime(runtime *Runtime) (err error) {
 		}
 	}
 
-	// Set up the lock manager
-	var manager lock.Manager
 	lockPath := DefaultSHMLockPath
 	if rootless.IsRootless() {
 		lockPath = fmt.Sprintf("%s_%d", DefaultRootlessSHMLockPath, rootless.GetRootlessUID())
 	}
-	if doRefresh {
+	// Set up the lock manager
+	manager, err := lock.OpenSHMLockManager(lockPath, runtime.config.NumLocks)
+	if err != nil {
```

Comment:
We should only do this if errors.Cause(err) == syscall.ENOENT

Otherwise it could be a different error - permission denied, lock count mismatch, etc